superset/viz.py

```diff
@@ -491,6 +491,18 @@ def data(self):
     def get_csv(self):
         df = self.get_df()
         include_index = not isinstance(df.index, pd.RangeIndex)
+        if config.get("CSV_EXPORT_USE_VERBOSE_NAMES_AS_HEADERS"):
+            d = {
```

Comment:
This dict should probably be called something more descriptive, e.g. `columns_with_verbose_names`

Reply:
Thanks, fixed.